packages/components/organisms/f-loyalty/src/components/Terms.vue

```diff
@@ -0,0 +1,36 @@
+<template functional>
+    <div :class="$style['c-loyalty-terms']">
+        <a
+            :class="$style['c-loyalty-terms-link']"
+            data-test-id="terms-and-conditions"
+            :href="parent.$t('termsUrl')">
+            {{ parent.$t("termsText") }}
```

Comment:
You could pass the translations down as props in order to avoid having to reference `parent`?

Reply:
Good idea I shall do that.